setup.md

```diff
@@ -54,14 +58,14 @@ data-toggle="tab">Linux</a></li>
       8. Close the Docker Desktop window
 4. Configure VS Code
    1. Open [this link](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.remote-wsl){:target="_blank"}{:rel="noopener noreferrer"}
-      to install the "Remote - WSL" extension for VS Code by clicking the `Install` button or by following the directions.
-   2. After installation, in VS Code choose "Open a Remote - WSL Window" and then "New WSL Window".
+      to install the **Remote - WSL** extension for VS Code by clicking the `Install` button or by following the directions.
+   2. After installation, in VS Code choose _Open a Remote - WSL Window_ and then _New WSL Window_.
 
       If you don't see those option, then press <kbd>Ctrl</kbd>+<kbd>Shift</kbd>+<kbd>P</kbd> and then type
-      "WSL" and they should appear at the top of the screen.
-   3. There should now be a second VS Code window that has "WSL: Ubuntu" in green at the lower left corner.
+      <kbd>WSL</kbd> and they should appear at the top of the screen.
```

Comment:
By the Carpentries style guide, this would mean a physical keyboard key labeled "WSL" ; maybe another type of emphasis is better?